SPECS/pcsc-lite/pcsc-lite.spec

```diff
@@ -1,16 +1,13 @@
-Name:           pcsc-lite
-Version:        1.9.0
-Release:        2%{?dist}
 Summary:        PC/SC Lite smart card framework and applications
-
+Name:           pcsc-lite
+Version:        1.9.5
+Release:        1%{?dist}
 License:        BSD
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://pcsclite.apdu.fr/
 Source0:        https://pcsclite.apdu.fr/files/%{name}-%{version}.tar.bz2
-Source1:        https://pcsclite.apdu.fr/files/%{name}-%{version}.tar.bz2.asc
-Source2:	gpgkey-F5E11B9FFE911146F41D953D78A1B4DFE8F9C57E.gpg
```

Comment:
Did we delete this file? I don't see it in the list of changed files.

Reply:
Deleted the .asc file. .gpg is not on github